apps/nuxt3-ssr/utils/harmonization.ts

```diff
@@ -75,7 +75,7 @@ const calcStatusForSingleVariable = (
     case "na":
       return "unmapped";
     case "partial":
-      return "complete";
+      return "partial";
```

Comment:
add or update test with case describing this path

Reply:
Added in the latest commit